Components/Avatar/avatar.css

```diff
@@ -1,25 +1,27 @@
+.avatar{
+  border-radius: 50%;
+  width: 3rem;
+  height: 3rem;
+}
+
 .avatar-xs {
-    border: 1px solid var(--theme-color);
     border-radius: 50%;
     width: 2rem;
     height: 2rem;
   }
   
   .avatar-sm {
-    border: 1px solid var(--theme-color);
     border-radius: 50%;
     width: 3rem;
     height: 3rem;
   }
   
   .avatar-md {
-    border: 1px solid var(--theme-color);
     border-radius: 50%;
     width: 6rem;
     height: 6rem;
   }
   .avatar-lg {
-    border: 1px solid var(--theme-color);
     border-radius: 50%;
```

Comment:
I can see you are using border-radius:50% everywhere
so why not make it a separate class?

Reply:
I got your point. But, avatar and floating icons are only things that we need border radius 50%. not any other(may be rounded images too). For Avatar we have a class, so, adding border-radius:50% directly to it, will be easy to use , rather than giving separate class every time.